### PR TITLE
README update to Docker instrucitons

### PR DESCRIPTION
**Pull request by exook**
Reviewed docker instructions to promote users to change "--mode" stead of copy-pasting the whole console block

Apptainer instructions not verified by me yet

### CODE REVIEW

**Comment by exook** [comment]
I actually agree with you Oliver, I'll revert bits of it :)

**Comment by exook** [comment]
So I agree with you to have a straight-forward copy+paste approach. So therefore I propose removing the code block containing the "base command" (I have personally copied that block and pasted it into my terminal)

I also think we should include `-rm` in the docker build command, so that our future master students don't fill their hard drives with docker images like I did.

what do you think?

File: documentation/setup/docker_setup.md [hunk 1]
```diff
@@ -1,105 +1,139 @@
 # Running with Docker #
+A setup for windows is available [here](documentation/setup/docker_setup_windows.md)
```
Comment (OliverWoolland) [inline documentation/setup/docker_setup.md:2]:
```suggestion
Setup instructions for windows are available [here](documentation/setup/docker_setup_windows.md)
```

File: documentation/setup/docker_setup.md [hunk 2]
```diff
@@ -1,105 +1,139 @@
...
+```console
+unzip baler.zip
+```
+Enter the root directory of baler
+```console
+cd baler
+```
+
+This process has created the following directory tree:
+```console
```
Comment (OliverWoolland) [inline documentation/setup/docker_setup.md:24]:
```suggestion
```console
tree
```

File: documentation/setup/docker_setup.md [hunk 3]
```diff
@@ -1,105 +1,139 @@
...
 --mount type=bind,source=${PWD}/data/,target=/baler-root/data \
-ghcr.io/uomresearchit/baler:latest 
-[--mode=... project--=...]
+pekman/baler:latest \
+--project=example_CFD \
+--mode=train
 ```
 
-Where:
+In this command, the "fixed" lines are:
```
Comment (OliverWoolland) [inline documentation/setup/docker_setup.md:65]: permanent? standard? fixed implies previously broken to me!

Comment (OliverWoolland) [reply documentation/setup/docker_setup.md:65]: common? base? really not sure but fixed doesn't hit me quite right

File: documentation/setup/docker_setup.md [hunk 4]
```diff
@@ -1,105 +1,139 @@
...
   * `docker run` invokes docker and specifies the running of a container
   * `-u ${UID}:${GID}` tells the container to use your username to create files
   * `--mount type=bind,source=${PWD}/projects/,target=/baler-root/projects` mounts the local (host) directory `./projects` to the container at `/projects`
-  * `ghcr.io/uomresearchit/baler:latest` specifies the container to run
-  
-Therefore the three commands detailed above become:
+  * `--mount type=bind,source=${PWD}/data/,target=/baler-root/data` mounts the local (host) directory `./data` to the container at `/data`
```
Comment (OliverWoolland) [inline documentation/setup/docker_setup.md:69]:
```suggestion
  * `--mount type=bind,source=${PWD}/data/,target=/baler-root/data` mounts the local (host) directory `./data` to the container at `/baler-root//data`
```

**Review by OliverWoolland — APPROVED** [review]
Approved, with a couple of comments giving suggestions

**Pull request merged**